src/main/java/org/jenkinsci/plugins/ghprb/GhprbPullRequestMerge.java

```diff
@@ -151,10 +150,10 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, final Build
             }
         }
 
-        Boolean isMergeable = cause.isMerged();
+        boolean isMergeable = cause.isMerged();
```

Comment:
`isMerged()` returns a `boolean`, not a `Boolean`.  So there's no possibility of NPE.